worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -34,6 +34,29 @@
 
 /**
  * An axis-aligned cuboid. It can be defined using two corners of the cuboid.
+ *
+ * Example usage:
+ * <pre>
+ * <code>
+ * Bukkit.getScheduler().runTaskTimerAsynchronously(this, () -> {
+ *    for (final Pair<Cuboid, List<WantedMaterial>> mine : mines.values()) {
+ *        final World world = BukkitAdapter.adapt(mine.getKey().getWorld());
+ *
+ *        final CuboidRegion selection = new CuboidRegion(
```

Comment:
This example usage doesn't actually explain anything beyond the constructor. Given these classes don't exist, it's not useful